.travis.yml

```diff
@@ -2,16 +2,18 @@ language: elixir
 elixir:
   - 1.3.1
   - 1.4.0
+  - 1.6.6
+  - 1.7.3
 otp_release:
   - 17.5
   - 18.0
   - 19.0
 before_script:
   - MIX_ENV=test mix do deps.get
 script:
-  - MIX_ENV=test mix test --no-start
+  - MIX_ENV=test mix test
 after_script:
-  - MIX_ENV=test mix coveralls.travis --no-start
+  - MIX_ENV=test mix coveralls.travis
 services:
   - redis-server
-sudo: false
```

Comment:
Why was this needed?

Reply:
There is an issue related to this: https://github.com/travis-ci/travis-ci/issues/7941
Redis seems to be broke if you keep `sudo: false`